axelrod/match.py

```diff
@@ -12,46 +12,70 @@ def is_stochastic(players, noise):
     of the players involved is stochastic."""
     return (noise or any(p.classifier['stochastic'] for p in players))
 
+
 class Match(object):
 
-    def __init__(self, players, turns, deterministic_cache=None, noise=0):
+    def __init__(self, players, turns, game=None, deterministic_cache=None,
+                 noise=0):
         """
         Parameters
         ----------
         players : tuple
             A pair of axelrod.Player objects
         turns : integer
                 The number of turns per match
+        game : axelrod.Game
+            The game object used to score the match
         deterministic_cache : dictionary
             A cache of resulting actions for deterministic matches
         noise : float
             The probability that a player's intended action should be flipped
         """
         self.result = []
-        self.players = list(players)
+        self.turns = turns
         self._classes = (players[0].__class__, players[1].__class__)
```

Comment:
I wasn't sure myself, so I left it well alone!!

I'm not certain when the players tuple was converted to a list or why.